mantidproject/mantid
# Improve workspace names for time sliced data

Improve output workspace names for time sliced data to include the time period rather than just a slice index.

**To test:**

- Open the ISIS Reflectometry GUI
- Enter two rows in the same group with run number 13460 and 13462. Set instrument to INTER.
- On the Event tab, select the second radio button and set the time slice to 500 secs
- On the runs tab, process the group.
- You should get 3 slices for run 13460 and 5 for 13462.
- Check that there are stitched workspaces for the first 3 slices.
- Check that you can plot all rows for the group from the GUI. It should plot all 3 for the first run and all 5 for the second.
- Check that you can plot the group. It should plot the 3 stitched workspaces.

Refs #21004 (partial fix)

**Release Notes** 
*Added to the release notes*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
